mcm/tools/locator.py

```diff
@@ -46,3 +46,18 @@ def cmsweburl(self):
             return 'https://cmsweb-testbed.cern.ch/'
         else:
             return 'https://cmsweb.cern.ch/'
+
+    def database_credentials(self):
+        """
+        Retrieve the credential's header to the CouchDB database.
+        It is taken from the runtime environment.
+
+        Returns:
+            str: CouchDB basic credentials coded as base64.
+        Raises:
+            RuntimeError: If they are not provided.
+        """
+        cred_header = os.getenv("COUCH_CRED", "")
+        if not cred_header:
+            raise RuntimeError("Please provide the CouchDB authentication header")
```

Comment:
Tell the user how it should do it in the error message:
```suggestion
            raise RuntimeError("Set COUCH_CRED to the CouchDB authentication header")
```